en/docs/learn/logging-in-to-opencart-using-the-identity-server.md

```diff
@@ -0,0 +1,131 @@
+service provider which can be configured to establish the trust between the plugin and WSO2 IS to securely authenticate 
+the user to the Opencart store.
+
+### The flow
+
+The diagram below demonstrates the flow of how OpenCart uses WSO2 Identity Server as a SAML2 federated authenticator to 
+authenticate a user.
+
+![opencart-is-flow](../assets/img/tutorials/opencart-is-flow.png)
+
+!!! tip "Before you begin!"
+    You need to have OpenCart installed. Refer: [https://docs.opencart.com/installation/](https://docs.opencart.com/installation/)
+
+Let's get started!
+
+## Configure OpenCart
+
+### Install SAML SSO extension
+
+1.  Visit the [OpenCart extension store](https://www.opencart.com/index.php?route=marketplace/extension) and download 
+the miniorange saml service provider extension.
+    ![opencart-extension-store](../assets/img/tutorials/opencart-extension-store.png)
+
+2.  Login to the OpenCart dashboard as administrator.
+    
+3.  Navigate to **Extensions > Installer** from the admin dashboard.
+    
+4.  Click on the **Upload** button and select the downloaded miniorange saml sp extension.
+    ![opencart-miniorange-install](../assets/img/tutorials/opencart-miniorange-install.png)
+    
+5.  Navigate to **Extensions > Extensions** and choose the extension type as **Modules**.
+    ![opencart-extensions-modules](../assets/img/tutorials/opencart-extensions-modules.png)
+
+6.  In the module list, you will see MiniOrange SAML SP. Click on the install button, [ + ] .
+
+### Configure the SAML SSO extension
+
+1.  Click on the **Edit** icon to start configuring the extension.
+    ![opencart-edit-miniorange](../assets/img/tutorials/opencart-edit-miniorange.png)
+    
+2.  Provide an application name.
+    
+3.  In the **Service Provider Metadata** tab, you will find the **SP Entity ID** and **ACS Url** which will be needed 
+later for identity provider configurations.
+    ![opencart-miniorange-sp-meta](../assets/img/tutorials/opencart-miniorange-sp-meta.png)
+    
+4.  In the **Identity Provider Setup** tab, provide the values for **Entity ID**, **Single Login URL** and 
+**SAML x509 Certificate**. These values should match with the SAML metadata values available in the identity provider.
+
+    !!! tip "Extract SAML metadata values"
+        
+        1. Login to WSO2 IS as the administrator.
+        2. Under the **Main** tab, select **Resident Identity Provider** under **Identity Providers**.
+        3. Expand the **Inbound Authentication Configuration** section.
+        4. Select **SAML2 Web SSO Configuration** and **Download SAML Metadata**.
+        5. The downloaded xml file contains the relevant information required for the identity provider setup.
+        
+    ![opencart-miniorange-idp](../assets/img/tutorials/opencart-miniorange-idp.png)
+    
+5.  Add the relevant IdP attributes by navigating to **Attribute Mapping** tab.
+    1. First name: http://wso2.org/claims/givenname
```

Comment:
Attribute can be given within `http://wso2.org/claims/givenname`
Apply wherever applicable.